cli/pom.xml

```diff
@@ -55,28 +55,10 @@
         <target>all</target>
     </properties>
 
-    <repositories>
-        <!--
-            Same as in the parent apache pom. Repeated here in case
-            direct parent's pom not found at relativePath. Can't fetch
-            the parent from remote repos for snapshot versions - no
-            snapshot repos enabled by default.
-        -->
```

Comment:
without this section, if you check out this project _only_ at a snapshot version i think it won't build because it won't find `brooklyn-parent:VVV-SNAPSHOT` (it's not in path and this won't know to look at the apache snapshot repo).

@neykov i suggest leaving this in, maybe expanding the comment above.  i think we want (and we say) projects can be built individually.